src/main/java/microsoft/exchange/webservices/data/util/TimeZoneUtils.java

```diff
@@ -0,0 +1,587 @@
+package microsoft.exchange.webservices.data.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.TimeZone;
+
+/**
+ * Miscellany timezone functions
+ */
+public class TimeZoneUtils {
+	//a map of olson name > Microsoft Name
+	final static private Map<String,String> olsonTimeZoneToMs = new HashMap<String,String>();
+	static {
+		olsonTimeZoneToMs.put("Africa/Abidjan", "Greenwich Standard Time");
```

Comment:
another option would be to put the static initializer directly to the HashMap.
```java
final static private Map<String,String> olsonTimeZoneToMs = new HashMap<String,String>(){
        {
                put("Africa/Accra", "Greenwich Standard Time");
                put("Africa/Addis_Ababa", "E. Africa Standard Time");
                put("Africa/Algiers", "W. Central Africa Standard Time");
                put("Africa/Asmara", "E. Africa Standard Time");
                [...]
        }
};
```
But your solution is also fine.